packages/@react-spectrum/dialog/stories/Dialog.stories.tsx

```diff
@@ -206,41 +207,45 @@ storiesOf('Dialog/Alert', module)
     })
   );
 
-function render({width = 'auto', ...props}) {
+function render({width = 'auto', isDismissable = undefined, ...props}) {
   return (
     <div style={{display: 'flex', width, margin: '100px 0'}}>
-      <DialogTrigger isOpen>
+      <DialogTrigger isDismissable={isDismissable} isOpen>
```

Comment:
Should the Dialogs in these stories be closable? If so the `isOpen` needs to change to `defaultOpen`